### PR TITLE
fix #745 - date time in different timezones

### LINKED ISSUE #745: Checklist/Date & Time: Haptics does not work properly
In the calendar view, haptics does not work properly. Please investigate. 
It might be just because of the screen resolution, but I tried several phones with no luck.
Behaviour to be seen in the enclosed video.
Example with date picker: I pick 20, but 19 gets marked.
With date: the haptics does not react to a cross (is does not remove the row.)

Android 14, Pixel 8

https://github.com/vexl-it/vexl/assets/25779015/be16c51a-daa4-4e25-bf22-9acad2646b75

### PR DESCRIPTION
**Pull request by kaladivo**

### CODE REVIEW

**Comment by kaladivo** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #786** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/786?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/786?utm_source=stack-comment).

File: apps/mobile/src/components/TradeChecklistFlow/components/DateAndTimeFlow/atoms.ts [hunk 1]
```diff
@@ -96,8 +97,12 @@ export function createTimeOptionAtomForTimeFromDropdown(
 export const handleAvailableDaysChangeActionAtom = atom(
   null,
   (get, set, day: DateData) => {
-    const dateTime = DateTime.fromMillis(day.timestamp)
-    const millis = fromDateTime(dateTime.startOf('day'))
+    const dateTime = DateTime.fromObject({
```
Comment (kaladivo) [inline apps/mobile/src/components/TradeChecklistFlow/components/DateAndTimeFlow/atoms.ts:100]: This is important!

**Review by SamTremko — APPROVED** [review]

File: apps/mobile/src/utils/unixMillisecondsToLocaleDateTime.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  return dateTime
+}
+
+export default function unixMillisecondsToLocaleDateTime(
+  milisec: UnixMilliseconds
+): DateTime {
+  if (!usersTimeZone) {
+    return DateTime.fromMillis(milisec)
+  }
+  console.log('Users time zone', usersTimeZone)
```
Comment (SamTremko) [inline apps/mobile/src/utils/unixMillisecondsToLocaleDateTime.ts]: Maybe just remove this comment

**Comment by kaladivo** [comment]
### Merge activity

* **Feb 1, 8:38 AM EST America/New_York**: @kaladivo merged this pull request with [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/786).

**Pull request merged**